docs/api/events.rst

```diff
@@ -865,6 +865,23 @@ Members
     :param after: The updated member's updated info.
     :type after: :class:`Member`
 
+.. function:: on_raw_presence_update(payload)
+
+    Called when a :class:`User` updates their presence.
```

Comment:
```suggestion
    Called when a member updates their presence.
```
It might be better not to link anything here, since there are no complete user/member objects being deserialized for this event.
The "user"/"member" phrasing could be discussed further, but personally I feel that "member" is more accurate here, given that there's one `presence_update` per mutual guild.

Reply:
Got it, thanks 6740164b